packages/broker/src/ConfigWizard.ts

```diff
@@ -225,59 +203,103 @@ export const selectDestinationPathPrompt = {
     if (!answers.parentDirExists) {
         mkdirSync(answers.parentDirPath)
     }
    
-    writeFileSync(answers.selectDestinationPath, JSON.stringify(config, null, 2))
-    chmodSync(answers.selectDestinationPath, '0600')
-    return answers.selectDestinationPath
+    writeFileSync(answers.selectStoragePath, JSON.stringify(config, null, 2))
+    chmodSync(answers.selectStoragePath, '0600')
+    return answers.selectStoragePath
 }
 
-export const startBrokerConfigWizard = async(): Promise<void> => {
+export const getNodeIdentity = (privateKey: string) => {
+    const nodeAddress = new Wallet(privateKey).address
+    const mnemonic = Protocol.generateMnemonicFromAddress(nodeAddress)
+    const networkExplorerUrl = `https://streamr.network/network-explorer/nodes/${nodeAddress}`
+    return {
+        mnemonic,
+        networkExplorerUrl
+    }
+}
+
+export const start = async(): Promise<void> => {
     try {
-        const answers = await inquirer.prompt(prompts)
-        const config = getConfigFromAnswers(answers)
-        const nodeAddress = new Wallet(config.ethereumPrivateKey).address
-        const mnemonic = Protocol.generateMnemonicFromAddress(nodeAddress)
+        const privateKeyAnswers = await inquirer.prompt(PRIVATE_KEY_PROMPTS)
+        const privateKey = getPrivateKey(privateKeyAnswers)
+        if (privateKeyAnswers.revealGeneratedPrivateKey) {
+            logger.info(`This is your node\'s private key: ${privateKey}`)
+        }
+        const pluginsAnswers = await inquirer.prompt(createPluginPrompts())
+        const config = getConfig(privateKey, pluginsAnswers)
+        const storageAnswers = await selectStoragePath()
+        const storagePath = await createStorageFile(config, storageAnswers)
         logger.info('Welcome to the Streamr Network')
-        logger.print(`Your node's generated name is ${mnemonic}.`)
-        logger.print('View your node in the Network Explorer:')
-        logger.print(`https://streamr.network/network-explorer/nodes/${nodeAddress}`)
-        logger.info('This is your node\'s private key. Please store it in a secure location:')
-        logger.alert(config.ethereumPrivateKey)
-        const storageAnswers = await selectValidDestinationPath()
-        const destinationPath = await createStorageFile(config, storageAnswers)
-        logger.info('Broker Config Wizard ran succesfully')
-        logger.print(`Stored config under ${destinationPath}`)
-        logger.print('You can start the broker now with')
-        logger.info(`streamr-broker ${destinationPath}`)
+        const {mnemonic, networkExplorerUrl} = getNodeIdentity(config)
+        logger.info(`Your node's generated name is ${mnemonic}.`)
+        logger.info('View your node in the Network Explorer:')
+        logger.info(networkExplorerUrl)
+        logger.info('You can start the broker now with')
+        logger.info(`streamr-broker ${storagePath}`)
     } catch (e) {
```

Comment:
Fix TypeScript warning:  the type of e is unknown (all 3 places where exception handling is done)

Reply:
tried grepping "catch (e" and it's type is either not given or `any` everywhere else, is the `any` typing here enough as well?

Reply:
Typing it `any` is fine